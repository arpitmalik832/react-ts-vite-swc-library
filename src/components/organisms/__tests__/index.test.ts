import '@testing-library/jest-dom';

import * as e from '../index';

describe('index.ts exports', () => {
  it('snapshot test', () => {
    expect(e).toMatchSnapshot();
  });

  it('should export Loader', () => {
    expect(e).toHaveProperty('Loader');
  });
});
