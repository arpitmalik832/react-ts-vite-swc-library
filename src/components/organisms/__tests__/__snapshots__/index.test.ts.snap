// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.ts exports snapshot test 1`] = `
{
  "Loader": [Function],
}
`;
